dashboard/app/views/pd/workshop_mailer/facilitator_pre_workshop.html.haml

```diff
@@ -18,9 +18,14 @@
       =link_to('facilitator landing page', 'https://code.org/educate/facilitator-landing/CSD') + '.'
     - elsif @workshop.course == 'Computer Science A'
       =link_to('facilitator landing page', 'https://code.org/educate/facilitator-landing/CSA') + '.'
+    - elsif @workshop.course == 'CS Fundamentals'
+      =link_to('facilitator landing page', 'https://code.org/educate/facilitator-landing/CSF') + '.'
   %li
     Have quick questions about resources, agendas, or participant requests? Post them in the facilitator
-    =link_to('Slack', 'https://codefacilitators6-12.slack.com/messages')
+    - if @workshop.course == 'CS Fundamentals'
+      =link_to('Slack', 'https://csffacilitators.slack.com/messages')
+    - else
+      =link_to('Slack', 'https://codefacilitators6-12.slack.com/messages')
```

Comment:
@dmcavoy Same question as above

Reply:
There is a K-5 facilitator and 6-12 facilitator slack channel. So there are 2